README.md

```diff
@@ -1,61 +1,20 @@
 # WizardsUi
 
-<a alt="Nx logo" href="https://nx.dev" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="45"></a>
+This workspace has been generated by:
+- **Angular v17**
+- **Storybook v7.6**
 
-✨ **This workspace has been generated by [Nx, a Smart, fast and extensible build system.](https://nx.dev)** ✨
+## Start the storybook
 
-## Start the app 
+To start the storybook run `npm run storybook`. Open your browser and navigate to http://localhost:4400/.
 
-To start the development server run `nx serve wizards-ui`. Open your browser and navigate to http://localhost:4200/. Happy coding!
+## Build the library
 
-## Generate code
+To build the library first change library version in libs\ui\package.json, then run `npm run build`, then go to dist\libs\ui and run `npm publish`
```

Comment:
Hmm, mozesz to rozbić na punkty np -,
albo coś aby bardziej była jako instrukcja a nie paragraf tekstu?